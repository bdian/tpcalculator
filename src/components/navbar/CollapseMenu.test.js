import React from 'react';
import CollapseMenu from './CollapseMenu';
import renderer from 'react-test-renderer';

test('Collapse Menu snapshot', () => {
  const component = renderer.create(
    <CollapseMenu />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});