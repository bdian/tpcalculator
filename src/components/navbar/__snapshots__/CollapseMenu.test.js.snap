// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Collapse Menu snapshot 1`] = `null`;
